modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc

```diff
@@ -36,48 +37,170 @@ Showcase apps demonstrate many features of {product-name}. To see these features
-|Database Server Port|*Port number of the PostgreSQL server, usually should be 5432*
-|Database Server Username|*Username of the PostgreSQL database connection*
-|Database Server Password|*Password of the PostgreSQL database connection*
-|Database name|*PostgreSQL database name*
+|Sync Application Name| syncshowcase
+|Sync Application Docker image| docker.io/aerogear/voyager-server-example-task
+|Sync Application Docker image version| latest
+|Sync App Port| 4000
+|Sync Application metrics endpoint| /metrics
+|Sync App GraphQL Endpoint| /graphql
+|Database Server Hostname| Hostname of the PostgreSQL server, usually *postgresql*
+|Database Server Port| Port number of the PostgreSQL server, usually *5432*
+|Database Server Username| *Username of the PostgreSQL database connection*
+|Database Server Password| *Password of the PostgreSQL database connection*
+|Database name| *PostgreSQL database name*
 |====
-.. xref:data-sync.adoc#sync-server-binding-keycloak[Protect the {sync-service} app using {keycloak-service}]
++
+. Provision the Mobile Developer Console
+.. Click `Browse Catalog` and select `Mobile` -> `Mobile Developer Console`.
+.. Complete the wizard to start the deployment of the *Mobile Developer Console*.
+
+[[creating-the-data-sync-mobile-app-binding]]
+=== Creating the Data Sync Mobile App binding
+
+[discrete]
+.Prerequisites
+
+xref:setting-up-mobile-services-to-demonstrate-showcase-apps[Setting up Mobile Services to Demonstrate Showcase Apps]
+
+[discrete]
+.Procedure
+
+. Log in to the Mobile Developer Console by clicking on the Mobile Developer Console link within Openshift.
+. Click *Create Mobile App*, and complete the wizard to create a Mobile App.
+. Click on the newly created *Mobile App* and click *Mobile Services*.
+. The Data Sync provisioned service should be available under the _Unbound Services_ heading. Click *Bind to App* and follow the wizard.
+. When the binding completes, click on *configuration*. The configuration for the Data Sync Mobile App is displayed, for example:
++
+[source,javascript]
+----
+{
+  "version": 1,
+  "namespace": "test12",
+  "clientId": "test16",
+  "services": [
+    {
+      "id": "8565c444-6bed-33e9-8616-0af08791569c",
+      "name": "sync-app-syncshowcase-test16",
+      "type": "sync-app",
+      "url": "https://sync-app-syncshowcase.testbed.com/graphql",
+      "config": {
+        "websocketUrl": "wss://sync-app-syncshowcasetestbed.com/graphql"
+      }
+    }
+  ]
+}
+
+module.exports = config;
+----
+
+[[configuring-the-showcase-app-to-point-at-the-provisioned-data-sync-service]]
+=== Configuring the Showcase App to point at the provisioned Data Sync service
+
+[discrete]
+.Prerequisites
+
+xref:creating-the-data-sync-mobile-app-binding[Creating the Data Sync Mobile App binding]
+
+[discrete]
+.Procedure
+
+1. Clone the showcase app
++
+include::../cloning-showcase-app.adoc[]
++
+2. Incorporate the Data Sync Mobile App code into the *mobile-services.js* file located within _src_:
```

Comment:
This was changed and ionic-showcase app is [now using `mobile-services.json`](https://github.com/aerogear/ionic-showcase/blob/master/src/mobile-services.json) as a configuration file

Reply:
https://github.com/aerogear/mobile-docs/pull/682/commits/5603aac4b52b0bcfd0b0bbe737866d1b653aa7bd